test/test_transformers.py

```diff
@@ -3979,12 +3981,27 @@ def test_is_causal_and_mask_fails(self, device):
         with self.assertRaisesRegex(ValueError, "CausalBias should not be used with causal=True"):
             scaled_dot_product_attention(query, key, value, attn_mask=attn_bias, is_causal=True, dropout_p=0.0)
 
+
+def build_and_import_openreg():
```

Comment:
This is usually done in run_test.py before running the tests that need it. Can we re-use the existing logic there to do this?
Also if we don't want this whole file to depend on the extension, we can move this test to a cpp_extension file

Reply:
Sure, I'll fix it.